common/build.gradle

```diff
@@ -1,6 +1,10 @@
 apply from: "$rootProject.projectDir/library.gradle"
 android {
     namespace 'com.revenuecat.purchases.common'
+
+    buildFeatures {
+        buildConfig true
```

Comment:
We use the `BuildConfig` file in the `common` module to enable debug logs automatically on debug versions.